internal/oidc/const.go

```diff
@@ -0,0 +1,22 @@
+package oidc
+
+const (
+	fallbackOIDCIssuer = "https://login.example.com:8080"
+	wellKnownPath      = "/.well-known/openid-configuration"
+	jwksPath           = "/.well-known/jwks.json"
+	authorizePath      = "/api/oidc/authorize"
+	tokenPath          = "/api/oidc/token" // nolint:gosec
+	userinfoPath       = "/api/oidc/userinfo"
+	consentPath        = "/api/oidc/consent"
+	introspectPath     = "/api/oidc/introspect"
+	revokePath         = "/api/oidc/revoke"
+)
+
+var scopeDescriptions = map[string]string{
```

Comment:
I like the idea of the descriptions. I don't remember if I did it or if you @james-d-elliott did it but I think we should think about a way to also declare additional ones in configuration. Maybe in a v2. I'm just sharing the idea here.

Reply:
Yeah I did this and mapped it for frontend in ConsentGET. I just added the standard ones I think we'll be supporting for now. It currently falls back to the scope name as the description.